src/components/authorPageComponents/ListOfArts/ListOfArts.js

```diff
@@ -1,13 +1,30 @@
 import React from 'react'
 
-import style from './ListOfArts.module.scss';
+import artsListStyle from './ListOfArts.module.scss';
 
 class ListOfArts extends React.Component {
     render() {
         return(
             <>
-            {/* this.props.artsArrayOfObject - this is an array of objects! */}
-            <p>{this.props.artsArrayOfObject[0].internal.content}....@list of arts</p>
+            <h3 className={artsListStyle.artsList_title}>Artist works</h3>
```

Comment:
You can also use bootstrap. Of the benefits - the cells are highlighted when hovering. The changes are minimal. [https://react-bootstrap.github.io/components/table/](https://react-bootstrap.github.io/components/table/)